README.md

```diff
@@ -1,2 +1,35 @@
 # starlight
 Cross-functional skill tracking system for Agile teams. 🌟
+
+## Installation
+
+Required packages:
+* git
+* Docker
+* docker-compose
+
+```bash
+git clone https://github.com/savoirfairelinux/starlight.git
+cd starlight
+cp .env.json.example .env.json
+python3 manage.py loaddata starlight/fixtures/fixtures.json
+docker-compose up --build
+```
+
+_Visit localhost:8000 in your browser_
+
+## Development workflow
+
+1. Create a super user for admin access
+
+```bash
+docker exec -it starlight_web1 python3 manage.py createsuperuser
+```
+
+2. Develop
+
+3. Apply migrations
+
+```bash
+docker exec -it starlight_web1 python3 manage.py makemigrations
```

Comment:
You can update these to use the same `docker-compose` format as the others.

The reason why it's preferable to use `docker-compose run` for these is that, that way, you can run the commands no matter what, otherwise it requires the process to already be running in another window or as a daemon.